src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppECSFargate.recipe

```diff
@@ -702,6 +702,15 @@
                             "Value": "Path"
                         }
                     ]
+                },
+                {
+                    "Id": "InternetFacing",
+                    "Name": "Internet-Facing",
+                    "Description": "Should the load balancer have an internet-routable address? Internet-facing load balancers can route requests from clients over the internet. Internal load balancers can route requests only from clients with access to the VPC.",
```

Comment:
nit: maybe change _"Should the load balancer have an internet-routable address?"_ -> _"Determines if the load balancer has an internet-routable address."_

Reply:
We do use a question for most other `"Bool"` options. For example, all the others in the Fargate recipe:
* "Do you want to create a new ECS cluster?"
* "Do you want to create a new role?"
* "Do you want to use the default VPC for the deployment?"
* "Do you want to create a new VPC?"
* "Do you want to create a new Load Balancer?"
* "Do you want to enable AutoScaling?"

I'm still inclined to match those for consistency, unless you feel strongly.

Reply:
I am fine with the wording if there is precedent for it👍